test/helpers/utils.js

```diff
@@ -73,14 +73,20 @@ export function getSigsWithVotes(_wallets, data, sigPrefix, maxYesVotes) {
 }
 
 export function encodeSigs(sigs = []) {
-  return Buffer.concat(sigs.map(s => ethUtils.toBuffer(s)))
+  return sigs.map(s => {
+    const buffer = [...ethUtils.toBuffer(s)]
+    return [
+      new BN(buffer.slice(0, 32)),
+      new BN(buffer.slice(32, 64)),
+      new BN(buffer.slice(64, 96))
+    ]
+  })
 }
 
-export async function checkPoint(wallets, proposer, stakeManager, { blockInterval = 1 } = {}) {
+export async function checkPoint(wallets, proposer, stakeManager, { blockInterval = 1, rootchainOwner } = {}) {
   const voteData = 'dummyData'
-  const sigs = ethUtils.bufferToHex(
-    encodeSigs(getSigs(wallets, ethUtils.keccak256(voteData)))
-  )
+  const sigs = encodeSigs(getSigs(wallets, ethUtils.keccak256(voteData)))
```

Comment:
This is required for heimdall as well even pos portal

Reply:
What is required?

Reply:
encoding the sigs right?

Reply:
Oh yeah, i see, will fix this moment